freeipa-client/src/main/java/com/sequenceiq/freeipa/client/operation/BatchOperation.java

```diff
@@ -0,0 +1,65 @@
+package com.sequenceiq.freeipa.client.operation;
+
+import static com.sequenceiq.freeipa.client.FreeIpaClient.METHOD_NAME_BATCH;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.BiConsumer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.freeipa.client.FreeIpaClient;
+import com.sequenceiq.freeipa.client.FreeIpaClientException;
+import com.sequenceiq.freeipa.client.FreeIpaClientExceptionUtil;
+import com.sequenceiq.freeipa.client.FreeIpaErrorCodes;
+
+public class BatchOperation extends AbstractFreeipaOperation<Object> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(BatchOperation.class);
+
+    private List<Object> operations;
+
+    private BiConsumer<String, String> warnings;
+
+    private Set<FreeIpaErrorCodes> acceptableErrorCodes;
+
+    private BatchOperation(List<Object> operations, BiConsumer<String, String> warnings, Set<FreeIpaErrorCodes> acceptableErrorCodes) {
+        this.operations = operations;
+        this.warnings = warnings;
+        this.acceptableErrorCodes = acceptableErrorCodes;
+    }
+
+    public static BatchOperation create(List<Object> operations, BiConsumer<String, String> warnings, Set<FreeIpaErrorCodes> acceptableErrorCodes) {
+        return new BatchOperation(operations, warnings, acceptableErrorCodes);
+    }
+
+    @Override
+    public String getOperationName() {
+        return METHOD_NAME_BATCH;
```

Comment:
is there a reason that BatchOperation uses a constant here? The other operations return a String directly.